src/index.ts

```diff
@@ -27,7 +27,8 @@ const PORT = process.env.PORT || 8000;
 const {
 	HUBSPOT_API_KEY: HAPIKEY,
 	SAASQUATCH_API_KEY: SAPIKEY,
-	SAASQUATCH_TENANT_ALIAS: STENANTALIAS
+	SAASQUATCH_TENANT_ALIAS: STENANTALIAS,
+  SERVER_TOKEN_SECRET: SERVER_TOKEN_SECRET,
```

Comment:
I added the token secret into my .env file does this need to be adjusted for the production build?

Reply:
Adding it to the Heroku config variables?

Reply:
Thanks added to Heroku 👍